diff -ruN vendor/danielbachhuber/composer-lock-updater/src/Checker.php vendor/danielbachhuber/composer-lock-updater/src/Checker.php
--- vendor/danielbachhuber/composer-lock-updater/src/Checker.php	2020-02-05 08:01:26.000000000 -0500
+++ vendor/danielbachhuber/composer-lock-updater/src/Checker.php	2021-11-05 10:23:24.209974318 -0400
@@ -9,6 +9,7 @@
 	 */
 	public static function check_executables( \stdClass $provider ) {
 		$execs = array_merge([
+			'bash',
 			'git',
 			'composer',
 			'tee',
@@ -21,7 +22,7 @@
 		}
 		Logger::info( 'Found required executables on system: ' . implode( ', ', $execs ) );
 
-		$output = shell_exec( 'set -o' );
+		$output = shell_exec( 'bash -c "set -o"' );
 		if ( false !== stripos( $output, 'pipefail' ) ) {
 			Logger::info( 'Found required pipefail option in shell.' );
 		} else {
diff -ruN vendor/danielbachhuber/composer-lock-updater/src/Runner.php vendor/danielbachhuber/composer-lock-updater/src/Runner.php
--- vendor/danielbachhuber/composer-lock-updater/src/Runner.php	2020-02-05 08:01:26.000000000 -0500
+++ vendor/danielbachhuber/composer-lock-updater/src/Runner.php	2021-11-05 10:23:33.896541441 -0400
@@ -253,7 +253,7 @@
 	 */
 	private function runComposerUpdate() {
 		$args = getenv( 'CLU_COMPOSER_UPDATE_ARGS' ) ? : '--no-progress --no-dev --no-interaction';
-		$cmd  = 'set -o pipefail && composer update ' . $args . ' 2>&1 | tee vendor/update.log';
+		$cmd  = 'bash -c "set -o pipefail && composer update ' . $args . ' 2>&1 | tee vendor/update.log"';
 		Logger::info( $cmd );
 		exec( $cmd, $output, $return_code );
 		if ( 0 !== $return_code ) {
